### PR TITLE
Add chat link

### PR DESCRIPTION
**Pull request by Elchi3**
I'd like to invite the BCD community to get closer to each other and closer to MDN folks. 
Mozilla has replaced its old IRC with matrix chat. We've created an MDN channel there. (I've used the "share room" functionality to get a link to the room).

### CODE REVIEW

**Review by jmswisher — CHANGES_REQUESTED** [review]
The "Share room" link takes you to a landing page that makes you select a Matrix client; even if you know which link to click, it then takes you to riot.im, not c.m.o. You then have to know to change your home server to mozilla.modular.im (not chat.mozilla.org). "Share room" does not actually do what it says. 

https://chat.mozilla.org/#/room/#mdn:mozilla.org is a more readable and usable version of the chat link, which takes you straight to the sign-in page for chat.mozilla.org. It still doesn't take you straight to the room, but at least it gets you to the right server.

**Comment by Elchi3** [comment]
Thank you @jmswisher! :+1:  I've updated this.

File: docs/contributing.md [hunk 1]
```diff
@@ -67,5 +67,4 @@ If the feature you're interested in is a JavaScript API, you can cross-reference
 
 ## Getting help
 
-If you need help with this repository or have any questions, contact the MDN team
-in the [#mdn](irc://irc.mozilla.org/mdn) IRC channel on irc.mozilla.org or write us on [discourse](https://discourse.mozilla-community.org/c/mdn).
+If you need help with this repository or have any questions, contact the MDN team on [chat.mozilla.org#mdn](https://chat.mozilla.org/#/room/#mdn:mozilla.org) or write us on [discourse](https://discourse.mozilla-community.org/c/mdn).
```
Comment (ddbeck) [inline docs/contributing.md]:
```suggestion
If you need help with this repository or have any questions, contact the MDN team on [chat.mozilla.org#mdn](https://chat.mozilla.org/#/room/#mdn:mozilla.org) or write to us on [Discourse](https://discourse.mozilla-community.org/c/mdn).
```

**Review by ddbeck — CHANGES_REQUESTED** [review]
The new link works great—I finally managed to join the chat. Some minor edits, but otherwise, this works for me now.

**Review by ddbeck — APPROVED** [review]

**Comment by Elchi3** [comment]
Thank you @ddbeck :+1:

**Review by redova87 — COMMENTED** [review]
>

**Comment by Kycilak** [comment]
Is the mozilla homeserver dead or are there just issues with certificates? It says to contact the admin but does not provide a contact so I am trying here.
Thanks

**Pull request merged**